packages/catalog-client/package.json

```diff
@@ -41,5 +41,6 @@
   },
   "files": [
     "dist"
-  ]
+  ],
+  "module": "dist/index.esm.js"
```

Comment:
same as above, what is this change required for?

Reply:
same as above